demos/jans-tarp/src/options/authFlowInputs.tsx

```diff
@@ -0,0 +1,337 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import InputLabel from '@mui/material/InputLabel';
+import Select from '@mui/material/Select';
+import LinearProgress from '@mui/material/LinearProgress';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import qs from 'qs';
+import axios from 'axios';
+import Utils from './Utils';
+import { v4 as uuidv4 } from 'uuid';
+import Stack from '@mui/material/Stack';
+import MenuItem from '@mui/material/MenuItem';
+import { IOption } from './IOption';
+import Alert from '@mui/material/Alert';
+import { ILooseObject } from './ILooseObject';
+
+const createOption = (label: string) => ({
+  label,
+  value: label,
+});
+
+export default function AuthFlowInputs({ isOpen, handleDialog, client }) {
+  const [open, setOpen] = React.useState(isOpen);
+  const [errorMessage, setErrorMessage] = React.useState("")
+  const [additionalParamError, setAdditionalParamError] = React.useState("")
+  const [displayToken, setDisplayToken] = React.useState(false);
+  const [acrValues, setAcrValues] = React.useState<readonly IOption[]>([]);
+  const [acrValueOption, setAcrValueOption] = React.useState("");
+  const [additionalParams, setAdditionalParams] = React.useState("");
+  const [loading, setLoading] = React.useState(false);
+  
+
+  React.useEffect(() => {
+    if (isOpen) {
+      handleOpen();
+    } else {
+      handleClose();
```

Comment:
try to reset all the values/errors on close if possible. As of now when I close the dialog and open it again, I can see the issuer error.

Reply:
done